recipes/commons_conf.rb

```diff
@@ -103,7 +104,7 @@
     path "#{node['openresty']['log_dir']}/*.log"
     enable true
     frequency 'daily'
-    rotate 7
+    rotate 2
```

Comment:
The default installation of nginx for Ubuntu and CentOS enforces a week's worth of logs. It should be left that way.